routes/account.route.js

```diff
@@ -1,10 +1,15 @@
 var express = require("express");
 const { checkAccessToken } = require("../middlewares/auth.middleware");
-const { account } = require("../controllers/account.controller");
-const { addAccountValidation } = require("../validators/user/account.validation");
- 
+const { account, withdrawal } = require("../controllers/account.controller");
+const {
+  addAccountValidation, updateAccountBalanceValidation,
+} = require("../validators/user/account.validation");
+
 var router = express.Router();
 
-router.route("/").post(checkAccessToken,addAccountValidation,account);
+router.route("/").post(checkAccessToken, addAccountValidation, account);
+router
+  .route("/:account_id/withdrawal")
```

Comment:
why only one route for withdrawal? will you create separate routes for withdrawal and deposit?